setup-kubectl-and-helm/action.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Setup kubectl and Helm.'
+description: 'Setup kubectl and Helm using kubeconfig file.'
+
+inputs:
+  kubeconfig:
+    description: 'The kubeconfig file content'
+    required: true
+  kubectl-version:
+    description: 'kubectl version'
+    required: false
+    default: 'latest'
```

Comment:
I wouldn't use the `latest` versions of Kubectl and Helm. It's very non-deterministic. It would be better to just make it required and force the CI to pin it. The versions of Kubectl and Helm should also be aligned with the cluster version to some extent.
https://helm.sh/docs/topics/version_skew/